doc/source/whatsnew/v0.25.0.rst

```diff
@@ -191,6 +191,8 @@ Optional libraries below the lowest tested version may still work, but are not c
 +-----------------+-----------------+
 | openpyxl        | 2.4.0           |
 +-----------------+-----------------+
+| pandas-gbq      | 0.10.0          |
++-----------------+-----------------+
```

Comment:
why are we requiring this? shouldn’t the older versions still work ok? this is very disruptive for users to have to upgrade

Reply:
The problem was that older versions don't know about `use_bqstorage_api`. I've updated the PR to populate a dictionary for new and deprecated kwargs only if they are explicitly set.

Reply:
of course but the point is that we cannot break prior installs

so you need to make things back compatible in pandas-gbq itself

we are not going to constantly upgrade the min supported version

Reply:
It is backwards compatible so long as we do as I've done in https://github.com/pandas-dev/pandas/pull/26104/commits/9581ddff72606fdc5171999904f06ce281053f9f

Or do you mean old versions of pandas-gbq should accept and ignore arbitrary `kwargs`?